pocket_coffea/utils/plot_utils.py

```diff
@@ -507,7 +525,13 @@ def _get_stacks(self, cat, spliteras=False):
                     for d in self.samples_mc
                 }
                 # create cycler from the colormap and instantiate it to get iterator
-                color = cycler("color", hep.styles.cms.cmap_petroff)()
+                cmap = CMAP_6 if len(self.nevents) <= 6 else CMAP_10
+                if len(self.nevents) > 10:
```

Comment:
If there are more than 10 processes to plot in the histogram there are colors assigned to multiple processes.
This is most likely not desired since you cannot tell them apart.
Is a warning enough in this case or should we handle this somehow differently?